src/app/loadout-builder/generated-sets/GeneratedSetItem.tsx

```diff
@@ -31,7 +31,7 @@ function EnergySwap({
 
   const modCost = _.sumBy(assignedMods, (mod) => mod.plug.energyCost?.energyCost || 0);
   const modEnergyHashNotAny = assignedMods?.find(
-    (mod) => mod.plug.energyCost?.energyType !== DestinyEnergyType.Any
+    (mod) => mod.plug.energyCost && mod.plug.energyCost.energyType !== DestinyEnergyType.Any
```

Comment:
Do we also want to filter out the inevitable manifest data bug where they make an item with "Solar energy cost 0"?

Reply:
Yeah can do, althought if they actually make a mod with an energy type and no energy it may be busted.